code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -40,7 +40,7 @@
 	icon_state = "deathsquad"
 	item_state = "swat_suit"
 	allowed = list(/obj/item/weapon/gun,/obj/item/ammo_box,/obj/item/ammo_casing,/obj/item/weapon/melee/baton,/obj/item/weapon/restraints/handcuffs,/obj/item/weapon/tank/internals)
-	armor = list(melee = 80, bullet = 80, laser = 50,energy = 50, bomb = 100, bio = 100, rad = 100)
+	armor = list(melee = 50, bullet = 40, laser = 30,energy = 50, bomb = 50, bio = 100, rad = 100)

```

Comment:
If death squads (and aliens) were more balanced we'd be able to play them
more often.

On 15 June 2015 at 19:10, KorPhaeron <notifications@github.com> wrote:

> In code/modules/clothing/spacesuits/miscellaneous.dm
> <https://github.com/tgstation/-tg-station/pull/9702#discussion_r32395638>:
>
> > @@ -40,7 +40,7 @@
> >  	icon_state = "deathsquad"
> >  	item_state = "swat_suit"
> >  	allowed = list(/obj/item/weapon/gun,/obj/item/ammo_box,/obj/item/ammo_casing,/obj/item/weapon/melee/baton,/obj/item/weapon/restraints/handcuffs,/obj/item/weapon/tank/internals)
> > -	armor = list(melee = 80, bullet = 80, laser = 50,energy = 50, bomb = 100, bio = 100, rad = 100)
> > +	armor = list(melee = 50, bullet = 40, laser = 30,energy = 50, bomb = 50, bio = 100, rad = 100)
>
> @GunHog <https://github.com/GunHog>
>
> They still basically kill anyone in 2 shots and admins can spawn infinite
> of them if the first wave dies somehow.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tgstation/-tg-station/pull/9702/files#r32395638>.
>